src/app.yaml

```diff
@@ -28,6 +28,7 @@ skip_files:
 - ^(.*/)?.*~$
 - ^(.*/)?.*\.py[co]$
 - ^(.*/)?.*/RCS/.*$
+- ^(.*/)?.*/jspm_packages/.*$
```

Comment:
This stops all of the stuff in the `/static/js/vendor/jspm_packages/` folder from going up on deployments, since we already make a self-executing bundle for the survey code.